storage/redis/redis.go

```diff
@@ -0,0 +1,183 @@
+package redis
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/lovoo/goka/storage"
+
+	redis "gopkg.in/redis.v5"
+)
+
+const (
+	offsetKey = "__offset"
+)
+
+type redisStorage struct {
+	client *redis.Client
+	hash   string
+}
+
+// NewRedis creates a new Storage backed by Redis.
+func NewRedis(client *redis.Client, hash string) (storage.Storage, error) {
```

Comment:
I would remove the Redis suffixes and prefixes from the types and methods as the package path and name implies this is a Redis storage for Goka. I think it would be cleaner to have `redis.New`.